ota4j-team/opentest4j
# Add TestFixture

## Overview
- Add `TestFixture`, `@Fixture` and `@ClassFixture`
- Add `TemporaryDirectory`
- Update project to require JDK 8 (note that Java SE 6 reached EOL back in 2013)

---

I hereby agree to the terms of the Open Test Alliance Contributor License Agreement.
